Library/Formula/gtk-mac-integration.rb

```diff
@@ -7,16 +7,18 @@ class GtkMacIntegration < Formula
   depends_on "pkg-config" => :build
   depends_on "gtk+"
   depends_on "gtk+3" => :recommended
+  depends_on "gobject-introspection"
```

Comment:
If you have `gtk+` installed, which this formula requires, you also have `gobject-introspection` installed, so there's no harm in using it.